### PR TITLE
jina3 migration for all examples [WIP]

### PR DESCRIPTION
**Pull request by joeyouss**

### CODE REVIEW

**Review by winstonww — APPROVED** [review]

File: basics/4_document_generators/app.py [hunk 1]
```diff
@@ -28,6 +28,6 @@
 with flow:
     flow.index(inputs=docs)
     query = Document(text=input("Please enter your search term: "))
-    response = flow.search(inputs=query, return_results=True)
+    response = flow.search(inputs=query)
```
Comment (winstonww) [inline basics/4_document_generators/app.py:31]: Might need to add `{'traversal_paths': 'r'}` in `uses_with` to get this to run in the meantime until spacy encoder is migrated to Jina 3. Same for example 5, 6 and 7

Comment (joeyouss) [reply basics/4_document_generators/app.py:31]: Ok!

**Comment by Shubhamsaboo** [comment]
@alexcg1 can you check this once?

**Comment by alexcg1** [comment]
Hey, let's do this as a branch to the repo, not as a fork. Forks we generally use for ppl outside the org

- Create a new repo branch like `migrate-v3` or something like that
- Open a PR request to merge your fork to that branch

**Pull request closed without merging**